packages/block-editor-tools/src/hooks/use-term/index.js

```diff
@@ -0,0 +1,18 @@
+import { store } from '@wordpress/core-data';
```

Comment:
Same here re typescript

Reply:
I'll leave this task to https://github.com/alleyinteractive/alley-scripts/issues/106